src/AbstractArray.php

```diff
@@ -638,4 +638,23 @@ protected function sortingKeys(array &$elements, $order = SORT_ASC, $strategy =
                 ksort($elements, $strategy);
         }
     }
+
+    /**
+     * Returns the first value that matches $func condition
+     *
+     * @param Closure $func
+     *
+     * @return mixed The resulting value
+     */
+    public function find(Closure $func)
```

Comment:
But what if you want to find the first _multiple of 2_ of the array? Do you don't know what it is before it's called, and use `filter` to this may be too much.